root/view/src/main/java/at/fhv/itb5c/view/user/UserViewController.java

```diff
@@ -0,0 +1,215 @@
+		
+		_userViewStates.put(ViewState.newState, new NewUserViewControllsController(this));
+		_userViewStates.put(ViewState.detailState, new DetailUserViewControlls(this));
+	}
+	
+	public void initialize() {
+		_firstNameTextField.textProperty().bindBidirectional(_userModel.getFirstName());
+        _lastNameTextField.textProperty().bindBidirectional(_userModel.getLastName());
+		_eMailTextField.textProperty().bindBidirectional(_userModel.getEMail());
+		_adressTextField.textProperty().bindBidirectional(_userModel.getAdress());
+        _telephoneNumberTextField.textProperty().bindBidirectional(_userModel.getTelephonenumber());   
+        _genderComboBox.setItems(FXCollections.observableArrayList(Gender.values()));   
+        _birthdayDatePicker.valueProperty().bindBidirectional(_userModel.getBirthDate());
+	
+        _typeOfSportCheckListView.setItems(FXCollections.observableArrayList(TypeOfSport.values()));
+        _typeOfSportCheckListView.getCheckModel().getCheckedItems().addListener(new ListChangeListener<TypeOfSport>() {
+			@Override
+			public void onChanged(javafx.collections.ListChangeListener.Change<? extends TypeOfSport> c) {
+				_userModel.setTypeOfSports(_typeOfSportCheckListView.getCheckModel().getCheckedItems());
+			}
+		});
+        
+        _memebershipFeeTextBox.textProperty().bindBidirectional(_userModel.getMemberShipFee().asObject(), new DecimalFormat());
+        _roleComboBox.setItems(FXCollections.observableArrayList(UserRole.values()));
+        
+        setState(ViewState.newState);
+	}
+	
+	@FXML
+	public void genderComboBoxOnActionEventHandler(ActionEvent event) {
+		if(_genderComboBox.getValue() != null) 
+		{
+			_userModel.setGender(_genderComboBox.getValue());
+		}
+	}
+	
+	@FXML
+	public void roleComboBoxOnActionEventHandler(ActionEvent event) {
+		if(_roleComboBox.getValue() != null) 
+		{
+			_userModel.setUserRole(_roleComboBox.getValue());
+		}
+	}
+
+	private PanelCloseHandler _panelCloseHandler;
+	
+	@Override
+	public void setPanelCloseHandler(PanelCloseHandler panelCloseHandler) {
+		_panelCloseHandler = panelCloseHandler;
+	}
+
+	public boolean saveModel() {
+		if(mandatoryFieldsSet()) {
+			return true;
+		} else {
+			return false;
+		}
+	}
+	
+	private boolean mandatoryFieldsSet() {
+		//refactor to validater -> string not empty validator ...
+		if((_userModel.getFirstName().getValue() != null) && (_userModel.getFirstName().getValue() != "") &&
+				(_userModel.getLastName().getValue() != null) && (_userModel.getFirstName().getValue() != "") &&
+				(_userModel.getAdress().getValue() != null) && (_userModel.getFirstName().getValue() != "") &&
+				(_userModel.getBirthDate().getValue() != null) && 
+				(_userModel.getGender() != null) &&
+				(_userModel.getUserRole() != null)) {
+			return true;
+		}
+		else {
+			return false;
+		}
+	}
+
+	public void close() {
+		_panelCloseHandler.close();
+	}
+	
+	private HashMap<ViewState, UserViewState> _userViewStates;

```

Comment:
Use the interface [`Map`](https://docs.oracle.com/javase/8/docs/api/java/util/Map.html)